File: FirstApp/Stock/Folder.vb
Imports FontAwesome.Sharp

Public Class Folder
    Inherits UserControl

    Private _backColor As Color
    Private _icon As FontAwesome.Sharp.IconChar

    <ComponentModel.Browsable(True)>
    Public Overrides Property BackColor As Color
        Get
            Return _backColor
        End Get
        Set(value As Color)
            BackPanel.BackColor = value
        End Set
    End Property

    <ComponentModel.Browsable(True)>
    Public Property Icon As FontAwesome.Sharp.IconChar
        Get
            Return _icon
        End Get
        Set(value As FontAwesome.Sharp.IconChar)
            Btn.IconChar = value
            _icon = value
            Invalidate()
        End Set
    End Property

End Class

File: OrnaTrack/Stock/UserControl/Folder.cs
using System.Drawing;
using System.Windows.Forms;
using FontAwesome.Sharp;

namespace FirstApp
{

    public partial class Folder : UserControl
    {

        private Color _backColor;
        private IconChar _icon;

        [System.ComponentModel.Browsable(true)]
        public override Color BackColor
        {
            get
            {
                return _backColor;
            }
            set
            {
                BackPanel.BackColor = value;
            }
        }

        [System.ComponentModel.Browsable(true)]
        public IconChar Icon
        {
            get
            {
                return _icon;
            }
            set
            {
                Btn.IconChar = value;
                _icon = value;
                Invalidate();
            }
        }

        public Folder()
        {
            InitializeComponent();
            BackColor = _BackColor;
            BackColor = _BackColor;
            BackColor = _BackColor;
            BackColor = _BackColor;
            BackColor = _BackColor;
        }

    }
}